CONTRIBUTING.md

```diff
@@ -80,6 +80,10 @@ Login: `devadventcalendar@gmail.com`
 
 Password: `P@ssw0rd`
 
+## Test environment
+
+Test environment is running on dev.devadventcalendar.pl with automatic deployment from develop_uat branch.
```

Comment:
please make `dev.devadventcalendar.pl` a link

Reply:
changed to link

Reply:
cannot see it here :)